pythran/pythonic/utils/meta.hpp

```diff
@@ -14,6 +16,7 @@ template<bool V0> struct all_of<V0> {
 };
 
 template<bool... V> struct any_of;
+template<> struct any_of<> : std::integral_constant<bool, true> {};
```

Comment:
It is not the case in Python and in C++. It seem to be more "logic" to me as none of them is "true".